Automattic/jetpack
# Remove body font normalization on contact-form module

This PR addresses an issue seen on classic editor, where the entire body style would be "normalized" to use sans-serif fonts. A user came complaining he was not happy with it. In the end, there is no reason why the module/package should normalize something at such a high level.

## Proposed changes:
Remove body font normalization from CSS files.

### Other information:


## Jetpack product discussion
p1677226003014399-slack-CBG1CP4EN

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Install classic editor plugin, have Jetpack plugin enabled. Create a post and verify the editor fonts remain as usual (default is some serif font Georgia, Times New Roman... )
